site/content/api/Credentials-API.md

```diff
@@ -0,0 +1,323 @@
++++
+title = "Credentials API"
+weight = 435
++++
+
+The *Credentials API* is used by *Protocol Adapters* to retrieve credentials used to authenticate *Devices* connecting to the adapter. In particular, the API supports the storage, look up and deletion of *shared secrets* which are often used by IoT devices by means of *username/password* based authentication schemes.
```

Comment:
"the API supports the storage, look up and deletion..." - update is missing, but is described below.